Makefile

```diff
@@ -404,20 +404,20 @@ bazel_test: failpoint-enable bazel_ci_prepare
 
 
 bazel_coverage_test: failpoint-enable bazel_ci_prepare
-	bazel $(BAZEL_GLOBAL_CONFIG) coverage $(BAZEL_CMD_CONFIG) \
+	bazel $(BAZEL_GLOBAL_CONFIG) --nohome_rc coverage $(BAZEL_CMD_CONFIG) \
```

Comment:
Why add `nohome_rc`? Seems bazel have `system_rc`, `workspace_rc` etc. Not familiar with bazel. Could you brief explain it?

Reply:
the ```.bazelrc``` which is saved at ```~/.bazelrc``` call ```home_rc``` in the bazel.

if you look the log in the CI. you will read this log.

```
INFO: Reading rc options for 'run' from /home/jenkins/.bazelrc:

  Inherited 'build' options: --local_ram_resources=22118 --local_cpu_resources=8

```

it is the homerc. but you also find this phenomenon.

```
[4,878 / 6,739] GoCompilePkg expression/expression_test.internal.recompileinternal.a; 134s remote-cache, linux-sandbox ... (146 actions, 86 running)

[5,055 / 6,739] GoCompilePkg expression/expression_test.internal.recompileinternal.a; 184s remote-cache, linux-sandbox ... (144 actions, 81 running)

[5,334 / 6,739] GoCompilePkg expression/expression_test.internal.recompileinternal.a; 241s remote-cache, linux-sandbox ... (125 actions, 70 running)

[5,494 / 6,739] GoCompilePkg expression/expression_test.internal.recompileinternal.a; 308s remote-cache, linux-sandbox ... (54 actions, 35 running)

[5,623 / 6,739] GoCompilePkg expression/expression_test.internal.recompileinternal.a; 384s remote-cache, linux-sandbox ... (172 actions, 95 running)

```

the number of action in bazel is more than the number of CPU. because the ```local_cpu_resources``` doesn't control the concurrency of ```gobuild```.

Reply:
it is why we need to disable the ```home_rc```